\documentclass{proposal}
\usepackage{epsfig}

% NSF proposal generation template style file.
% based on latex stylefiles  written by Stefan Llewellyn Smith and
% Sarah Gille, with contributions from other collaborators.

\newcommand{\degrees}{$\!\!$\char23$\!$}


\renewcommand{\refname}{\centerline{References cited}}

% this handles hanging indents for publications
\def\rrr#1\\{\par
\medskip\hbox{\vbox{\parindent=2em\hsize=6.12in
\hangindent=4em\hangafter=1#1}}}

\def\baselinestretch{1}

\begin{document}

\begin{center}
{\Large{\bf Project Summary}}\\*[3mm]
{\bf Proposal Title} \\*[3mm]

Peter Acu\~na Combs \\

\end{center}

\noindent
{\bf Proposed Fellowship Activities}

Here I propose conducting a tightly focused series of experiments to answer key questions in Drosophila tracheal development. The earliest steps in the development of this tubular epithelial network have become a model for branching morphogenesis, a process also relevant in human development and disease. I will first address whether tracheal precursors each have a unique transcriptional cellular identity, or whether morphological or physiological differences are solely acquired at later developmental time points. Next, I will compare tracheal development in a range of Drosophila species, with the final aim to assay tracheal cell types across those species, and seek correlations with cis-regulatory sequence change upstream of tracheal genes.

\noindent
{\bf Intellectual Merit}

My research up until this point has uniquely qualified me to undertake this research program. This experiment in Aim 1 is a natural extension of my goals of characterizing genome-wide changes in expression across spatially varying structures.  Furthermore,  I have extensive familiarity with existing comparative experiments that were conducted in my PhD lab. This will directly support the work in Aims 2 and 3. The entire project will leverage my computational skills, creating an exceptionally tight feedback loop between experiment and analysis. The study will further help me to generalize my findings on early patterning to the development of complex tissues.

\noindent
{\bf Broader Impacts}

While the proposed work will take place abroad, I ultimately intend to return to the United States.  Science is a truly international undertaking, and developing a globally competitive workforce involves forming international contacts and collaborations. While improving the conditions at home for scientists of underrepresented minorities is important, it is critical to also allow minorities to seek safe spaces without the same historical issues surrounding minority participation.  My ultimate goal is to pursue a a career in the biotech industry, probably via the route of initially leading an academic research group and creating a spin-off which also helps support the NSF's aim of increasing partnerships between academia and industry. 

% strong statement, this minority sentence... ...Acuna... Hispanic?
% Yep, Hispanic.  This particular NSF offering is specifically to ``broaden participation of under-represented groups in biology.''  I have a thoroughly upper-middle class background, but

\ \\


\renewcommand{\thepage} {B--\arabic{page}}

\newpage

% reset page numbering to 1.  This is helpful, since the text can only
% be 6 pages, and reviewers will want to believe we've kept within
% those limits

\pagenumbering{arabic}
\renewcommand{\thepage} {D--\arabic{page}}

\newpage


\noindent{\Large \bf PROJECT DESCRIPTION}

\section{Introduction}


Symmetry breaking is a key principle in metazoan development.  Organisms begin with a single undifferentiated cell, but ultimately must specify a large number of morphologically, positionally, and functionally distinct tissues. Often, equivalent neighboring cells must therefore make different decisions, mediated by a combination of external signals, intercellular communication and stochastic effects.

The instructions for this symmetry breaking are encoded in the genome, but as yet we lack a detailed understanding of how cells respond to these cues to reliably differentiate into the appropriate cell types located in the appropriate places. Given that the distribution of growth factors etc. follows the laws of physics, the rate of successful development into a fully functional adults suggests that at least some of these processes are robust or self-correcting. While a number of models of transcriptional response to cues have been mooted, (NOT SURE WHAT REFS), we still lack a lot of data how this cell specific robustness is ultimately leading to healthy tissue. So while starting from a set of very similar cellular characteristics, it is not clear to what extent each of these structures maintains a distinct identity.

Unlike the vertebrate plan of a central circulatory system to provide oxygen for the whole animal, flies perfuse their organs directly using a number of trachea distributed throughout the body.  The specification, determination and branching of tracheal cells in the Drosophila embryos provides an excellent system to study the specification of repeated structures in development.  At approximately stage 10 (~5 hours after egg-lay), about 40 cells of the dorsal ectoderm form a placode in each metamere on both sides of the embryo. These cells undergo exactly one round of mitosis. Following these initial events, until the end of embryogenesis, this set of $80 \times 10 \times 2 = 1600$ cells proceed to invaginate into the underlying tissue. Most of their migratory behavior is in response to FGF (branchless) driven cues, but EGF and BMP signalling confer spatial information for the correct localization of tracheal branches. While a number of studies of mutants have established the primary factors in the genetic network that directs tracheal formation (see Ghabrial et al., 2003, for review), it has not been possible to fully characterize the transcriptional identity of early tracheal tissues. (There's this rubbish Debbie Andrews paper "Trachealess regulates all tracheal genes", but it's conceptionally so flawed it's not even science - check it out).

Beyond elucidating general principles in branching and differentiation, studying the Drosophila embryo is likely to provide useful insights into branching morphogenesis in human tissue development in particular.  Genes associated with lung disease in humans are often homologous to the network in flies (that's unpublished so far -- but you could cite the papers about Sprouty that was first discovered in the tracheal system but is hugely important in lung development as well).  Drosophila embryos are naturally transparent, inexpensive to rear, have short generation times. Each of these allows more rapid experimental iteration.  Furthermore, the small, relatively simple genome should ease the parsing of genetic networks.  MAYBE worth to mention the about 300 publications on tracheal development, the ~1000 genes known to be expressed in the trachea on the basis of GO or BDGP annotation, and our overall understanding of the "network" (like the ones set into context to each other in the Ghabrial review).


\section{Aim: Determine whether tracheal branches are molecularly distinguishable}

Previously, it has not been practical to perform genome-wide screens for genes with differential function across different tracheal branches.  In the first part of this aim, I will leverage my experience performing RNAseq or Fluidigm measurements on small samples to dissect and profile the transcriptomes of individual tracheal placodes, which consist of ~80 cells at their earliest detectable stage.  While I will initially direct my attention to the transcription factors known to be involved in the orchestration of tracheal gene expression programs (Trachealess, Tango, Ventral veins lacking, Stat92E, etc), I am ultimately interested in the entire repertoire of expressed genes. 

I will dissect cells from tracheal placodes from embryos at three time points: from earliest identifiable placodal structures at stage 11, cells from each of the extensions in the five finger stage in stage 12 (following FGF stimulation) and prior to the fusion of differentiated and branched trachea, with approximately half-hour time resolution.  If different tracheal tubes acquire distinct identities at any time after the initial stage, this simple time series will suggest mechanisms for the specification. Using known binding sites upstream of the distinguishing genes, I will use genome bioinformatics to establish a list of likely transcription factors that drive the differentiation process, which can then be directly tested using a number of traditional and state-of-the-art genetic techniques.

In addition to profiling the transcriptome of the dissected tracheal placodes, I will also measure RNA levels in the tissues from which they were excised.  I will verify that my dissections are both complete and specific by ensuring that levels of both primary markers (like trachealess) as well as secondary markers (like btl, sms, rho, and tdf) are high in the placodes and low in the surrounding tissue via in situ hybridization of similar dissections before proceeding to full transcriptome libraries.

Regardless of whether all tracheal placodes share the same transcriptional program, it is possible that there are additional phosphorylation differences that potentially affect the activity, even given the same protein complement.  The Adryan lab is based in the Cambridge Systems Biology Centre is well positioned to explore whether these differences also exist, since it has ongoing collaborations with the research group of Prof. Kathryn Lilley and access to proteomic mass-spec facilities and well-developed expertise in performing and interpreting these assays (CITE).


\section{Aim: Characterize the evolution of tracheal development}

The Drosophila genus is extremely broad, encompassing more than 40 million years of differentiation, and adaptation to both restricted ecological niches and cosmopolitan distributions. (Not an expert, but mention that this is roughly equivalent than the difference between man and chicken). There has not, as yet, been a comprehensive morphological study of tracheal development across these species.  In this aim, I will perform time lapse microscopy of tracheal development in a broad range of Drosophila species, using CRISPR/CAS9 directed integration of a GFP tag in the trachealess locus. If feasible, I will also use the 2A12 marker or chitin binding probe for the comparative visualization of tracheal lumens.


While time-lapse imaging of Drosophila development has been performed, the primary focus has been on establishing equivalent developmental time points \cite{Kuntz:2014bc}.  Consequently, the dataset does not have tracheal markers or sufficient numbers of individuals to address variation in tracheal number.  I will therefore conduct similar time-lapse series, but focusing on a narrower window during which tracheal structures form.  This will give the benefit of increasing effective throughput, allowing for robust estimation of the variance in size, speed, and degree of branching. 

My hypothesis is that key tracheal qualities, such as number of placodes, degree of branching, and size of tracheal tubes will co-associate with the original habitat of each particular species, even when reared under identical conditions.  The primary function of trachea is to provide oxygen to tissues, and the bioavailability of that oxygen will depend on altitude and temperature. If there is any plasticity to the traits studied, the rapid generation time of Drosophila should aid selection in rapidly providing optimal levels of oxygenation, without wasting resources developing overly intricate tracheal structures. 


In collaboration with John Pool at the University of Wisconsin, the Adryan group is in possession of D. melanogaster fly strains endemic at SOMEHEIGHT, and it will be interesting to see if there are any longterm morphological as well as transcriptional adjustments.

\section{Aim: Probe the evolution of tracheal developmental transcriptomes with an eye towards understanding regulation}
Nature has performed an extensive mutational assay, with strong selection at every generation towards functional development.  Examining the 12 sequenced Drosophila genomes, and correlating changes around known tracheal cis-regulatory modules (of which there are zero to one...).

I will perform RNAseq experiments on dissected trachea from three of the original set of 12 sequenced Drosophila species: D. melanogaster, D. yakuba and D. pseudoobscura.
% (I'm thinking of time and cost here. Any "better" Drosophilas you'd think?)

By comparing changes in putative cis-regulatory sequence around genes that do and do not change expression levels across the Drosophila clade, I will help address issues around cis-regulatory grammars (CITE). EXPAND HOW YOU'RE GOING TO IDENTIFY CIS-REG SEQUENCES.

Developing tracheal tissue is the ideal sample to examine for this type of experiment.  Projects such as modENCODE, which have done some transcription profiling in other Drosophila species, have looked at samples that consist of a large number of already differentiated tissue types (typically whole embryos or adults), effectively averaging many very different transcriptional signals (CITE).  By contrast, this experiment would focus on a single tissue type, which maximizes signal-to-noise on the genes that are actually changing across evolution.  Furthermore, the choice of a tissue in the process of differentiation will help address the narrower question of what changes are responsible for establishment of cell fate, which is likely to be different from maintenance (CITE). 

This aim will build upon and complement the results from Aim 1.  Having a relatively complete inventory of transcription factors in D. melanogaster will provide a very well annotated baseline.   This experiment will also feed back into the transcriptome profiling genes with a coherent pattern of conservation across the phylogeny are more likely to be functionally important to the process of differentiation.

 In analyzing this data, I will consider not only genes that are well conserved, but those that seem to be rapidly evolving differential levels of expression.  The most interesting correlate is likely to be the original environment of the fly.  For instance, species that evolved at higher altitudes are likely to require higher levels of branching to compensate for the lower oxygen levels (CITE).  While actual oxygen levels during development are likely to contribute, coherent changes in gene expression levels even when reared at the same oxygen saturation would suggest that species evolve a particular set point, which individuals can adjust according to the environment (CITE). 


\section{Time Line and Management Plan}

% BIG LOL. So unrealistic. :-)
The proposed project is ambitious, but is feasible to complete within the three years of funding requested.  Assuming a start date of September 1, an approximate timeline of the project is: 

\begin{description}
\item[September -- November 2015] Dissection of trachea in {\em D. melanogaster} embryos.  

\item[December 2015 -- February 2016] Refinement of time-lapse imaging conditions. 

\item[December 2015 -- March 2016] Analysis of wild-type RNAseq data. 

\item[January -- July 2016] Time-lapse imaging in 12 species at ambient oxygen concentrations

\item[April -- May 2016] Preparation of manuscript detailing results from Aim 1.

\item[June -- December 2016] 

\item[August -- December 2016] Time-lapse imaging at high and low oxygen concentrations.


\item[]  

\end{description}



\section{Summary:  Significance of proposed work}

\subsection{Intellectual Merit}

\subsection{Broader Impacts}



\newpage
\pagenumbering{arabic}
\renewcommand{\thepage} {E--\arabic{page}}

\bibliography{draft}
\bibliographystyle{jponew}


\end{document}
